daemon/emer-machine-id-provider.c

```diff
@@ -186,9 +187,18 @@ emer_machine_id_provider_class_init (EmerMachineIdProviderClass *klass)
 static void
 emer_machine_id_provider_init (EmerMachineIdProvider *self)
 {
+  g_autoptr(GError) local_error = NULL;
   EmerMachineIdProviderPrivate *priv =
     emer_machine_id_provider_get_instance_private (self);
 
+
+  if (!write_tracking_id_file (TRACKING_ID_OVERRIDE, &local_error))
```

Comment:
This would cause the ID to be reset every time the system reboots. We don't want that – we want the ID to persist unless/until the user re-sets it. It should be written only if missing.